README.md

```diff
@@ -1,6 +1,33 @@
 # magmaviz
 
-CThis package contains four different data visualization functions with the magma theme for EDA.
+Exploratory Data Analysis is one of the key steps in a machine learning project. This package aims to make this process easy by providing python functions based on the 'Altair' package to plot four common types of plots with the magma color scheme. To maximize interpretability, the plots have defined color schemes (discrete, diverging, sequential) based on the kind of data they show.
+
+## Functions
+
+The four data visualization functions included in the package are outlined below:
+
+### Boxplot
+
+Returns a boxplot based on the data frame, a numerical feature to view the distribution of and a categorical feature to bucket data into categories. Additionally, there is a boolean option to facet the boxplots into separate charts.
+
+### Correlation Plot
```

Comment:
Should we add the function call with the arguments under the header?

Reply:
I think we can leave as-is for now. I believe the function calls can go under USAGE once we have them ready.

Reply:
Makes sense, I agree that this is an overview of the functions in the  package